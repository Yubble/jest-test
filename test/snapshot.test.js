/**
 * @Name: 
 * @Description: 
 * @Author: 刘燕保
 * @Date: 2020-11-17 15:14:23
**/

// 测试snapshot快照功能
const { conf1, conf2 } = require('./snapshot.conf')

test("测试 conf1 对象", () => {
  expect(conf1).toMatchSnapshot()
})

// test("测试 conf2 对象", () => {
//   expect(conf2)
// })
