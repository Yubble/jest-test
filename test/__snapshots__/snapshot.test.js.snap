// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`测试 conf1 对象 1`] = `
Object {
  "port": 8081,
  "server": "http://localhost",
}
`;
